src/React.AspNet/HtmlHelperExtensions.cs

```diff
@@ -5,26 +5,26 @@
  *  This source code is licensed under the BSD-style license found in the
  *  LICENSE file in the root directory of this source tree. An additional grant
  *  of patent rights can be found in the PATENTS file in the same directory.
- */
-
+ */
+
 using React.Exceptions;
-using React.TinyIoC;
-
+using React.TinyIoC;
+
 #if LEGACYASPNET
 using System.Web;
 using System.Web.Mvc;
-using IHtmlHelper = System.Web.Mvc.HtmlHelper;
+using IHtmlHelper = System.Web.Mvc.HtmlHelper;
 #else
 using Microsoft.AspNet.Mvc.Rendering;
 using IHtmlString = Microsoft.AspNet.Html.Abstractions.IHtmlContent;
 #endif
-
+
 #if LEGACYASPNET
-namespace React.Web.Mvc
+namespace React.Web.Mvc

```

Comment:
I can't see any noticeable change in this line so it's probably replaced newlines (`\r\n` with `\n` or vice versa). Could you please revert all the lines that aren't directly related to your change?